src/MetaOptions.js

```diff
@@ -0,0 +1,28 @@
+import changeCaseObject from 'change-case-object'
+
+const BODY_KEY_CASE_OPTIONS = {
+  SNAKE_CASE: changeCaseObject.snakeCase,
+  CAMEL_CASE: changeCaseObject.camelCase,
+  PARAM_CASE: changeCaseObject.paramCase
+}
+
+export default class MetaOptions {
+  constructor(options) {
+    if (!options.bodyKeyCase || !this.isValidBodyKeyCase(options.bodyKeyCase)) {
+      throw new Error('This body key formatting option is not allowed.')
+    }
+
+    this.bodyKeyCase = options.bodyKeyCase
+  }
+
+  isValidBodyKeyCase(bodyKeyCase) {
```

Comment:
```suggestion
  isValidBodyKeyCase(bodyKeyCase) {
	  Object.keys(BODY_KEY_CASE_OPTIONS).indexOf(bodyKeyCase) != -1
  }
```

this should return a boolean